src/parser/hunter/survival/modules/spells/ButcheryCarve.tsx

```diff
@@ -46,13 +47,13 @@ class ButcheryCarve extends Analyzer {
 
   get avgTargetsHitThreshold() {
     return {
-      actual: (this.targetsHit / this.casts).toFixed(1),
+      actual: Number((this.targetsHit / this.casts).toFixed(1)),
```

Comment:
Potentially `+((this.targetsHit / this.casts).toFixed(1))` instead; do we have a preference?

Reply:
I think Number() is more telling than +() is